packages/cli/src/handlers/download.ts

```diff
@@ -213,7 +214,7 @@ export const downloadHandler = async (
         );
     }
 
-    const projectId = config.context.project;
+    const projectId = config.context.project || options.project;
```

Comment:
I don't think this is the right way to get the projectId, it should be the other way arround, the  `options` should override the context if provided.

Reply:
I see.

Fixed: https://github.com/lightdash/lightdash/pull/13185/commits/427f9e97d3a2462af62e23c95ae99d11deb16372